docker/main/Dockerfile

```diff
@@ -30,6 +30,9 @@ RUN --mount=type=tmpfs,target=/tmp --mount=type=tmpfs,target=/var/cache/apt \
     --mount=type=cache,target=/root/.ccache \
     /deps/build_nginx.sh
 
+# https://github.com/felipecrs/cgroup-scripts/commits/master/get_cpus.sh
+ADD --link --chmod=755 "https://github.com/felipecrs/cgroup-scripts/raw/178c3ed0cb55465af6652fbd2b419573ccadea8a/get_cpus.sh" /usr/local/nginx/
```

Comment:
I think we would prefer just have the script directly inside the run file to reduce external dependencies and this docker build change. Would be good to leave the commented link so it is clear where it came from

Reply:
Hm... are you sure? I'll be a little ugly. 😅

Reply:
yes 👍